utils/build-script-impl

```diff
@@ -2326,10 +2333,17 @@ for host in "${ALL_HOSTS[@]}"; do
                 )
 
                 if [[ ! "${SKIP_BUILD_LIBICU}" ]] ; then
+                    LIBICU_BUILD_DIR="$(build_directory ${host} libicu)"
+                    ICU_TMPINSTALL=${LIBICU_BUILD_DIR}/tmp_install
                     cmake_options=(
                         "${cmake_options[@]}"
                         -DSWIFT_PATH_TO_LIBICU_SOURCE:PATH="${LIBICU_SOURCE_DIR}"
-                        -DSWIFT_PATH_TO_LIBICU_BUILD:PATH="$(build_directory ${host} libicu)"
+                        -DSWIFT_PATH_TO_LIBICU_BUILD:PATH="${LIBICU_BUILD_DIR}"
+                        -DSWIFT_${SWIFT_HOST_VARIANT_SDK}_${SWIFT_HOST_VARIANT_ARCH}_ICU_UC_INCLUDE:STRING="${ICU_TMPINSTALL}/include"
+                        -DSWIFT_${SWIFT_HOST_VARIANT_SDK}_${SWIFT_HOST_VARIANT_ARCH}_ICU_I18N_INCLUDE:STRING="${ICU_TMPINSTALL}/include"
+                        -DSWIFT_${SWIFT_HOST_VARIANT_SDK}_${SWIFT_HOST_VARIANT_ARCH}_ICU_STATICLIB:BOOL=TRUE
+                        -DICU_UC_LIBDIR:PATH="${LIBICU_BUILD_DIR}/lib"
+                        -DICU_I18N_LIBDIR:PATH="${LIBICU_BUILD_DIR}/lib"
```

Comment:
Hmm, does this work?  I thought that it requires that the path to the library is given instead.  I think it would be nicer if we just used `-DSWIFT_${SWIFT_HOST_VARIANT_SDK}_${SWIFT_HOST_VARIANT_ARCH}_ICU_{UC,I18N}=`

Reply:
I just kept these the same as they were before.  Maybe they were always dead?